system/libraries/javascript/Jquery.php

```diff
@@ -1,40 +1,110 @@
-<?php  if ( ! defined('BASEPATH')) exit('No direct script access allowed');
-
+<?php
```

Comment:
@henriquepesi Can you tell me why are you touching files in system/* ?

Reply:
@gurungrahul2 sorry, I think I've made a mistake with some  files, I've sent a new PR, please ignore this PR above

Reply:
@henriquepesi I can't find your new PR.